content_scripts/VirtualCamera.js

```diff
@@ -578,7 +578,12 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
             // tween the matrix every frame to animate it to the new position
             // let cameraNode = realityEditor.sceneGraph.getSceneNodeById('CAMERA');
             let currentCameraMatrix = realityEditor.gui.ar.utilities.copyMatrix(this.cameraNode.localMatrix);
-            let newCameraMatrix = tweenMatrix(currentCameraMatrix, realityEditor.gui.ar.utilities.invertMatrix(destinationCameraMatrix), 0.3);
+            let destinationCameraMatrix = realityEditor.gui.ar.utilities.invertMatrix(newCameraLookAtMatrix);
+            let totalDifference = sumOfElementDifferences(destinationCameraMatrix, currentCameraMatrix);
+            if (totalDifference < 0.00001) {
+                return; // don't animate the matrix with an infinite level of precision, stop when it gets very close to destination
+            }
+            let newCameraMatrix = tweenMatrix(currentCameraMatrix, destinationCameraMatrix, 0.3);
```

Comment:
Is this where it would turn off tweening for first person mode? Or is that somewhere with the velocity

Reply:
Just realized I forgot to include that feature. Pushed a new commit with those lines, they indeed go here